pims/norpix_reader.py

```diff
@@ -150,12 +183,17 @@ def _verify_frame_no(self, i):
         if i >= self._image_count or i < 0:
             raise ValueError("Frame number is out of range: " + str(i))
 
+    def set_process_func(self, process_func):
```

Comment:
I think that `pims.pipeline` (derived from Slicerator) already provides equivalent functionality. Could users just use that instead?